src/commands/index.ts

```diff
@@ -1,21 +1,10 @@
 import * as vscode from 'vscode';
 import { getConfig } from '../utils/vscode-extensions';
 import { LanguageResourceManager } from '../i18n';
-import {
-  IconGenerator,
-  mergeConfig,
-  schema,
-  toggleAngularPreset,
-  toggleOfficialIconsPreset,
-  toggleHideFoldersPreset,
-} from '../icon-manifest';
+import * as iconManifest from '../icon-manifest';
```

Comment:
The `import` list was getting a bit long so I figured I switch to the `allias` convention.